easybuild/easyconfigs/c/cuDNN/cuDNN-5.0.eb

```diff
@@ -0,0 +1,32 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+easyblock = 'Tarball'
+
+name = 'cuDNN'
+version = '5.0'
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for
+    deep neural networks."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Nvidia developer registration required.
+# Download link: https://developer.nvidia.com/rdp/cudnn-download
+sources = ['%(namelower)s-7.5-linux-x64-v%(version)s-rc.tgz']
```

Comment:
done, thanks @boegel. Actually I added 5.0 as I saw it on the web site but didn't notice it was a RC. I just installed it on our production system and it seems to be OK (see my latest commit).